sensor/common/filesystem/pipeline/pipeline.go

```diff
@@ -0,0 +1,175 @@
+package pipeline
+
+import (
+	"context"
+
+	sensorAPI "github.com/stackrox/rox/generated/internalapi/sensor"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/protocompat"
+	"github.com/stackrox/rox/pkg/uuid"
+	"github.com/stackrox/rox/sensor/common/clusterentities"
+	"github.com/stackrox/rox/sensor/common/detector"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+type Pipeline struct {
+	detector detector.Detector
+	stopper  concurrency.Stopper
+
+	accessChan      chan *storage.FileAccess
+	clusterEntities *clusterentities.Store
+
+	msgCtx context.Context
```

Comment:
I'm not seeing this being used anywhere. Are you planning on using it in a follow-up?

Reply:
Yes. This will be used to wire this up to the detector, where the following comment is currently
```
// TODO: Send event to detector
```